Dockerfile.jupyter

```diff
@@ -0,0 +1,17 @@
+FROM jupyter/base-notebook:d0b2d159cc6c
+
+USER $NB_USER
+
+RUN pip install sparkmagic
+
+RUN mkdir /home/$NB_USER/.sparkmagic
+COPY sparkmagic/example_config.json /home/$NB_USER/.sparkmagic/config.json
+RUN sed -i 's/localhost/spark/g' /home/$NB_USER/.sparkmagic/config.json
+RUN jupyter nbextension enable --py --sys-prefix widgetsnbextension
+RUN jupyter-kernelspec install --user $(pip show sparkmagic | grep Location | cut -d" " -f2)/sparkmagic/kernels/pyspark3kernel
```

Comment:
Yeah, so R is not set up properly for Livy. Let's disable installing the SparkR kernel.